packages/meter/src/Meter.ts

```diff
@@ -65,8 +72,20 @@ export class Meter extends SizedMixin(ObserveSlotText(SpectrumElement, '')) {
     protected override render(): TemplateResult {
         return html`
             <sp-field-label size=${this.size} class="label">
-                ${this.slotHasContent ? html`` : this.label}
-                <slot>${this.label}</slot>
```

Comment:
For reference, this code pattern supports the following...

By default it applies `this.label` as the default content of the slot, meaning the following will have a visible label of `This is a meter`.

```html
<sp-meter label="This is a meter"></sp-meter>
```

However, a `<slot>` will be filled _ANY_ nodes that are available as a child of the element, so _NONE_ of these will be labeled similarly, if ONLY a `<slot>` is leveraged:

```html
<sp-meter label="This is a meter"> </sp-meter>
<sp-meter label="This is a meter">
</sp-meter>
<sp-meter label="This is a meter">

</sp-meter>
```

This is why we use:

```html
${this.slotHasContent ? html`` : this.label}
<slot>${this.label}</slot>
```

In the case that there is no actual content in the `<slot>` but there is an empty node in the light DOM that would prevent the default `<slot>` content, this will ensure that the visible label is present for all above cases. When there is actual content in the light DOM:

```html
<sp-meter label="This is a meter">
    This has a special label.
</sp-meter>
```

Neither the default `<slot>` content, or the fallback helper will be available, so the only visible content will be `This has a special label.`

That means Meter and Progress Bar likely need to persist this pattern, while Progress Circle, which does not currently support a visible label, likely does not need it.